docs/design/supported_tracepoints.md

```diff
@@ -652,11 +647,10 @@ graph LR
   source_timestamp_sub --> dispatch_subscription_callback
 ```
 
-プロセス内通信では publish されたメッセージのアドレスで紐づけを行う。  
-プロセス間通信では source_timestamp で紐づけを行う。
+As well as flow of message transmission, a message is identified by `message_addr` for intra-process communication, but `source_timestamp` for inter-process communication.
 
-callback_start から message_addr/source_timestamp までが一意に紐付けられる。  
-また、publisher 側のグラフで示した通り、message_addr/source_timestamp から publish までが一意に紐付けられる。  
-したがって、publish から callback_start までは一意に紐付けることができる。
+`message_addr` and `source_timestamp` is mapped a corresponding `callback_start`.  
+As explained, `message_addr` and `source_timestamp` are identifier for transmission flow using `publish` method.
+This means that a invoked `publish` method is mapped to a `callback_start`.
 
-ただし、コールバックの実行と publish の紐付けはできていないので注意。
+However this does not mean that CARET can map a `callback_start` to a corresponding `publish`. We can trace a certain flow from `publish` to `callback_start`, but the reversed flow is not out of CARET's capability.
```

Comment:
Could yout add following sentences?

> The mapping between callback_start and publish will be improved in v0.3.* release or later.

Reply:
Fixed in https://github.com/tier4/CARET_doc/pull/40/commits/e70eda1881e8fc8214f745c3bda1aeea170ae96b